airwallex/src/main/java/com/airwallex/android/view/PaymentMethodsActivity.kt

```diff
@@ -73,6 +73,7 @@ class PaymentMethodsActivity : AirwallexCheckoutBaseActivity(), TrackablePage {
     }
 
     override fun onBackButtonPressed() {
+        AirwallexLogger.info("PaymentMethodsActivity onBackButtonPressed")
```

Comment:
Is it possible to move all code in this function to `AirwallexActivity` so it's reusable?